exporting/prometheus/remote_write/README.md

```diff
@@ -0,0 +1,38 @@
+# Prometheus remote write exporting connector
+
+## Prerequisites
+
+To use the Prometheus remote write API with [storage
+providers](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage), install
+[protobuf](https://developers.google.com/protocol-buffers/) and [snappy](https://github.com/google/snappy) libraries.
+Next, re-install Netdata from the source, which detects that the required libraries and
```

Comment:
Could you also re-install using a kickstart script, or does it have to be from source?

Reply:
AFAIK `kickstart.sh` doesn't install the dependencies needed for the remote write connector.